src/Panel.vue

```diff
@@ -338,10 +340,17 @@
             this.insertCaretInsideHeader(this.$refs.headerWrapper.innerHTML);
         }
       });
-      const panelHeader = this.$slots.header ? this.$refs.headerWrapper.innerHTML : this.headerContent;
-      const panelHeaderText = jQuery(panelHeader).wrap('<div></div>').parent().find(':header').text();
-      if (panelHeaderText) {
-        this.$refs.cardContainer.setAttribute('id', panelHeaderText.trim().replace(/\s+/g, '-').toLowerCase());
+
+      if (this.headerContent) {
+        const panelHeaderText = jQuery(this.headerContent).wrap('<div></div>').parent().find(':header').text();
+        if(panelHeaderText) {
+          this.$refs.cardContainer.setAttribute('id', string(panelHeaderText).slugify().toString());
+        }
+      } else if(this.$refs.headerWrapper.innerHTML) {
+        const header = this.$refs.headerWrapper.innerHTML).wrap('<div></div>').parent().find(':header');
```

Comment:
This line doesn't compile (it should be `jQuery(this.$refs.headerWrapper.innerHTML)`?)